packages/templates/cra-template-web-viewer/template/.env

```diff
@@ -4,6 +4,7 @@ IMJS_AUTH_CLIENT_REDIRECT_URI = ""
 IMJS_AUTH_CLIENT_LOGOUT_URI = ""
 IMJS_AUTH_CLIENT_SCOPES ="imodelaccess:read imodels:read realitydata:read"
 IMJS_AUTH_AUTHORITY="https://ims.bentley.com"
+IMJS_URL_PREFIX=""
```

Comment:
remove, i dont want to expose this to users

Reply:
removed